loculus_values/values.yaml

```diff
@@ -932,68 +954,78 @@ defaultOrganismConfig: &defaultOrganismConfig
         example: ncov-tools
         displayName: Quality control method name
         header: Sequencing
       - name: qualityControlMethodVersion
         ontology_id: GENEPIO:0100558
         definition: The version number of the method used to assess whether a sequence passed a predetermined quality control threshold.
-        guidance: Methods updates can make big differences to their outputs. Provide the version of the method used for quality control. The version can be expressed using whatever convention the developer implements (e.g. date, semantic versioning). If multiple methods were used, record the version numbers in the same order as the method names. Separate the version numbers using a semi-colon.
+        guidance: If multiple methods were used, record the version numbers in the same order as the method names. Separate the version numbers using a semi-colon.
         example: "1.2.3"
         displayName: Quality control method version
         header: Sequencing
       - name: qualityControlDetermination
         ontology_id: GENEPIO:0100559
-        definition: The determination of a quality control assessment.
-        guidance: Select a value from the pick list provided. If a desired value is missing, submit a new term request to the PHA4GE QC Tag GitHub issuetracker using the New Term Request form.
+        definition: The determination (result) of a quality control assessment.
+        #guidance: Select a value from the pick list provided. If a desired value is missing, submit a new term request to the PHA4GE QC Tag GitHub issuetracker using the New Term Request form.
         example: sequence failed quality control
         displayName: Quality control determination
         header: Sequencing
       - name: qualityControlIssues
         ontology_id: GENEPIO:0100560
         definition: The reason contributing to, or causing, a low quality determination in a quality control assessment.
-        guidance: Select a value from the pick list provided. If a desired value is missing, submit a new term request to the PHA4GE QC Tag GitHub issuetracker using the New Term Request form.
+        #guidance: Select a value from the pick list provided. If a desired value is missing, submit a new term request to the PHA4GE QC Tag GitHub issuetracker using the New Term Request form.
         example: low average genome coverage
         displayName: Quality control issues
         header: Sequencing
       - name: qualityControlDetails
         ontology_id: GENEPIO:0100561
         definition: The details surrounding a low quality determination in a quality control assessment.
-        guidance: Provide notes or details regarding QC results using free text.
         example: CT value of 39. Low viral load. Low DNA concentration after amplification.
         displayName: Quality control details
         header: Diagnostics
```

Comment:
I also think all these values should either be removed or marked as no input as we perform additional quality control on samples and do not use QC that users upload.